storagemarket/nodes.go

```diff
@@ -105,6 +105,10 @@ type StorageProviderNode interface {
 
 	// GetProofType gets the current seal proof type for the given miner.
 	GetProofType(ctx context.Context, addr address.Address, tok shared.TipSetToken) (abi.RegisteredSealProof, error)
+
+	// IsUnsealed checks if we have an unsealed sector file for the given sector which contains the unsealed deal data
```

Comment:
Follow-up: Why was this added? It appears to be unused in this code.

Reply:
Good catch 👍 

Removed in https://github.com/filecoin-project/go-fil-markets/pull/576/commits/c6cd9ae4259fb3eb477dfd7f091f7c8432aec63a